chainstate/test-framework/src/random_tx_maker.rs

```diff
@@ -700,10 +781,56 @@ impl<'a> RandomTxMaker<'a> {
                             }
                         }
 
-                        TxOutput::Transfer(
-                            OutputValue::Coin(new_value),
-                            Destination::AnyoneCanSpend,
-                        )
+                        let destination =
+                            key_manager.new_destination(self.chainstate.get_chain_config(), rng);
+                        match rng.gen_range(0..5) {
+                            0 => TxOutput::LockThenTransfer(
+                                OutputValue::Coin(new_value),
+                                destination,
+                                OutputTimeLock::ForBlockCount(rng.gen_range(0..5)),
```

Comment:
How did you pick the `0..5` interval? Should it be made configurable or a constant?

Reply:
I would prefer to put this in a constant and explain above the constant where this number came from. Same for all other magic numbers.